oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/DocumentNodeStoreBuilder.java

```diff
@@ -326,7 +326,7 @@ public Set<String> getFullGCIncludePaths() {
     }
 
     public T setFullGCExcludePaths(@Nullable String[] excludePaths) {
-        if (isNull(excludePaths) || excludePaths.length == 0) {
+        if (isNull(excludePaths) || excludePaths.length == 0 || Arrays.equals(excludePaths, new String[]{""})) {
```

Comment:
I would extract this `new String[]{""}` as a constant field, e.g EMPTY_STRING_ARRAY